bluetooth/README.md

```diff
@@ -167,6 +172,91 @@ substring(cod::bit(24) from 17 for 6) as minor_device_class,
 substring(cod::bit(24) from 12 for 5) as major_device_class
 ```
 
+As a part of updating/modernizing Bluetooth Reader Tables [ISSUE #196](https://github.com/CityofToronto/bdit_data-sources/issues/196), the following four tables are prepared.  
+* `reader_history`
+* `reader_locations`
+* `routes`
+* `reader_status_history`
```

Comment:
I know this is supposed to be introducing the new tables you've created, but this comment doesn't make sense where it is, which is under the `ClassOfDevice` heading.
This should be placed under the `Internal Tables`, and maybe move the sections for the individual tables above the `ClassOfDevice` section